### PR TITLE
some small fixes 🤪

### PR DESCRIPTION
**Pull request by polylymer**
### This pull request includes:

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] I have read the [CONTRIBUTING](../CONTRIBUTING.md) document.
- [x] My code follows the code style of this project.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### Changes made to the repository:

- fixed a overflowing DropDownButton using [relative_scale](https://pub.dev/packages/relative_scale) flutter package
- added missing nameSplitter
- fixed overflowing login page
- added url links for prerequisites in readme

### CODE REVIEW

File: lib/feature/tasks/task_setup_page.dart [hunk 1]
```diff
@@ -360,16 +360,22 @@ class _TaskSetupPageState extends State<TaskSetupPage> {
             children: [
               Text(t.page.tasks.setup.question.task_environment),
               Container(
+                height: 40,
```
Comment (TheMeinerLP) [inline lib/feature/tasks/task_setup_page.dart:363]: What happend if you have a lager device or smaller one ?

**Comment by TheMeinerLP** [comment]
The rest of it, LGTM

File: lib/feature/tasks/task_setup_page.dart (hunk 1 above)
Comment (polylymer) [reply lib/feature/tasks/task_setup_page.dart:363]: oh, i didn't test that. I'll fix it tomorrow

File: lib/feature/tasks/task_setup_page.dart [hunk 2]
```diff
@@ -360,15 +361,23 @@ class _TaskSetupPageState extends State<TaskSetupPage> {
             children: [
               Text(t.page.tasks.setup.question.task_environment),
               Container(
+                height: (MediaQuery.of(context).size.height * ((30 / 100) / 5.3333333333)).roundToDouble(),
```
Comment (TheMeinerLP) [inline lib/feature/tasks/task_setup_page.dart:364]: Where you get this formula ?

**Review by TheMeinerLP — APPROVED** [review]

File: lib/feature/tasks/task_setup_page.dart (hunk 2 above)
Comment (polylymer) [reply lib/feature/tasks/task_setup_page.dart:364]: I found it in the [relative scale](https://pub.dev/packages/relative_scale) flutter package

Comment (TheMeinerLP) [reply lib/feature/tasks/task_setup_page.dart:364]: You are welcome to use it to fix that 😉

**Comment by TheMeinerLP** [comment]
LGTM

**Pull request merged**